apps/vs-code-react/src/app/designer/customEditorService.tsx

```diff
@@ -0,0 +1,52 @@
+import type { IEditorParameterInfo, IEditorProps, IEditorService } from '@microsoft/logic-apps-shared';
+import { Link } from '@fluentui/react';
+
+export interface CustomEditorServiceOptions {
+  areCustomEditorsEnabled?: boolean;
+  openRelativeLink?: (relativeLink: string) => void;
+}
+
+export class CustomEditorService implements IEditorService {
+  _areCustomEditorsEnabled = true;
+
+  constructor(public readonly options: CustomEditorServiceOptions) {
+    const { areCustomEditorsEnabled } = options;
+    this._areCustomEditorsEnabled = areCustomEditorsEnabled || true;
```

Comment:
NIT: this is always going to be true, not matter what you send.  I wonder if the CustomEditorsEnabled is needed

Reply:
I'm following the format in existing customEditorService in LocalDesigner, would this be unnecessary?

Reply:
I noticed that you changed it to `??` which makes more sense and it works fine.
What I meant with the original comments was that 
`this._areCustomEditorsEnabled = false || true;` its equals to true.
`this._areCustomEditorsEnabled = false ?? true;` its equals to false.